include/rocksdb/sst_file_reader.h

```diff
@@ -24,11 +24,17 @@ class SstFileReader {
   // Prepares to read from the file located at "file_path".
   Status Open(const std::string& file_path);
 
-  // Returns a new iterator over the table contents.
+  // Returns a new iterator over the table contents as a DB iterator.
```

Comment:
Maybe explain what DB iterator means (only the latest value of each key is returned). Similarly for raw table iterator below.